pkg/tui/sources/syslog/syslog.go

```diff
@@ -64,10 +64,8 @@ func (m syslogCmdModel) Cmd() string {
 	syslogKeys := [5]string{"address", "protocol", "cert", "key", "format"}
 
 	for _, key := range syslogKeys {
-		if inputs[key] != "" {
```

Comment:
what makes this guard no longer necessary?

Reply:
Moved the check to the `GetInputs` func in `textinputs.go`